client/src/views/ScorecardView.vue

```diff
@@ -72,9 +69,12 @@ export default defineComponent({
 });
 </script>
 
-<style scoped>
+<style scoped lang='scss'>
+@import '../assets/styles/global.scss';
+@import '@blueconduit/copper/scss/01_settings/design-tokens';
+
 .actions-to-take {
-  background-color: #E1F5FE;
+  box-shadow: 0px 0px 8px #757575;
```

Comment:
Would one of [the shadows from Copper](https://github.com/BlueConduit/copper/blob/main/packages/copper/scss/01_settings/_design-tokens.scss#L66) work here?

Reply:
Obsolete!